pkg/virt-config/config_test.go

```diff
@@ -482,4 +482,18 @@ var _ = Describe("ConfigMap", func() {
 		emulation = clusterConfig.IsUseEmulation()
 		Expect(emulation).To(BeFalse())
 	})
+
+	table.DescribeTable("when feature-gate", func(openFeatureGates string, isLiveMigrationEnabled, isSRIOVLiveMigrationEnabled bool) {
+		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.FeatureGatesKey: openFeatureGates},
+		})
+
+		Expect(clusterConfig.LiveMigrationEnabled()).To(Equal(isLiveMigrationEnabled))
```

Comment:
I could not find `isLiveMigrationEnabled` implementation, what is it?

Reply:
`isLiveMigrationEnabled` and `isSRIOVLiveMigrationEnabled` are boolean that passed to the `DescribeTable` function (line 486)